app/views/components/docs/contents-list.yml

```diff
@@ -0,0 +1,100 @@
+name: Contents list
+description: Lists a page’s contents with links pointing to headings within the document
+body: |
+  Pass a list of contents each with an `href` and `text`. The `href` should point at the ID of a heading within the page.
+
+  Supports nesting contents one level deep, currently only used by specialist documents. When nesting the top level list items
+  display in bold.
+
+  `format_numbers` option will pull out numbers in the link text to render them as though they were the list style type. Applies to
+  numbers at the start of text, with or without a decimal. See the [format complex numbers fixture](/component-guide/contents-list/formats_complex_numbers) for details.
+accessibility_criteria: |
+  The contents links must:
```

Comment:
Should this be 'contents list'? Or are you writing criteria for links within the list? If so, maybe 'contents list links' might make that clearer.